persistent-mongoDB/Database/Persist/MongoDB.hs

```diff
@@ -878,6 +878,8 @@ filterToBSON fname v filt = case filt of
     showFilter In = "$in"
     showFilter NotIn = "$nin"
     showFilter Eq = error "EQ filter not expected"
+    showFilter Like = error "LIKE filter not expected"
+    showFilter NotLike = error "NOT LIKE filter not expected"
```

Comment:
We should be able to support the `Like` query in MongoDB: https://stackoverflow.com/questions/3305561/how-to-query-mongodb-with-like